src/bottom-sheet/bottom-sheet.ts

```diff
@@ -136,8 +136,20 @@ export class BottomSheet extends LitElement {
     }
   };
 
+  private dispatchCloseEvent() {
+    this.dispatchEvent(
+      new CustomEvent('close', {
+        bubbles: true,
+        composed: true,
+      }),
+    );
+  }
+
   private handleDismiss = (): void => {
     this.disappear = true;
+    if (this.dispatchCloseEvent) {
+      this.dispatchCloseEvent();
```

Comment:
the method is useless. 

```ts
this.dispatchEvent(
      new CustomEvent('close', {
        bubbles: true,
        composed: true,
      }),
    );
```

is good enough. no need to define a method to wrap an event.